README.md

```diff
@@ -82,54 +77,40 @@ Here is an example of how to use the method:
 
 ``` js
 var winston = require('winston');
-var {flushLogsAndExit} = require('winston-loggly-bulk');
+var { flushLogsAndExit } = require('winston-loggly-bulk');
 
 winston.log("info", "Hello World from Node.js!");
 flushLogsAndExit();
-  
 ```
 
 ## Motivation
+
 `tldr;?`: To break the [winston][1] codebase into small modules that work together.
 
 The [winston][1] codebase has been growing significantly with contributions and other logging transports. This is **awesome**. However, taking a ton of additional dependencies just to do something simple like logging to the Console and a File is overkill.
 
 ## Installation
 
-### Installing npm (node package manager)
-
-``` bash
-  $ curl http://npmjs.org/install.sh | sh
-```
-
-### Installing winston-loggly-bulk
-
-If you are running npm version 3 or higher then run the below command to setup the logging-
-
 ``` bash
-  $ npm install winston-loggly-bulk
+  npm install winston-loggly-bulk
 ```
 
-If you are running npm version 2 or lower then run the below command to setup the logging-
-
-``` bash
-  $ npm install winston-loggly-bulk winston
-```
-* __Note:__ To check the currrent npm version run the below command-
+Note: If you are using npm version 2, please run the below command:
 
 ``` bash
-  $ npm -v
+  npm install winston-loggly-bulk winston
 ```
 
 ## Run Tests
-All of the winston tests are written in [vows][6], and cover all of the use cases described above. You will need to add valid credentials for the various transports included to test/config.json before running tests:
+
+All of the winston tests are written in [vows][5], and cover all of the use cases described above. You will need to add valid credentials for the various transports included to `test/config.json` before running tests:
 
 ``` js
   {
     "transports": {
       "loggly": {
         "subdomain": "your-subdomain",
-        "inputToken": "really-long-token-you-got-from-loggly",
+        "token": "really-long-token-you-got-from-loggly",
         "auth": {
```

Comment:
This part appears to be pointless? It's not used by the tests. Maybe someone planned to login to loggly API and verify the logs arrived as part of the tests... but there's no such thing implemented. Let's remove it from here and from the sample config.

Reply:
Yes, pointless. Anyway, the package uses node-loggly-bulk and there is auth used, so I decided not to remove it. If it's useless node-loggly-bulk should be reviewed as well.